MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/utils/JsonLoader.java

```diff
@@ -0,0 +1,52 @@
+package com.mapbox.mapboxandroiddemo.utils;
+
+import android.os.AsyncTask;
+import android.support.annotation.Nullable;
+
+import java.io.IOException;
+import java.lang.ref.WeakReference;
+
+import timber.log.Timber;
+
+/**
+ * This class is a helper class to asynchrously load data
+ * while holding a weak reference to a callback.
+ * T data that is loaded
+ * @param <T> data to be loaded
+ */
+public class JsonLoader<T> extends AsyncTask<Void, Void, T> {
```

Comment:
class can be removed

Reply:
Gah, keep forgetting to delete that damn unused class. Still seeing weird behavior, so it's still WIP

![ezgif com-resize (1)](https://user-images.githubusercontent.com/4394910/58275868-e5ebd280-7d4a-11e9-9261-52562c64c416.gif)

Reply:
that issue is https://github.com/mapbox/mapbox-gl-native/issues/14715, feel free to ignore this bug

Reply:
Awesome 🚀